[2022-02-19 16:22:30,700] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: demo_etl.Load_Data.Save_to_DB manual__2022-02-19T16:21:55.210923+00:00 [queued]>
[2022-02-19 16:22:30,864] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: demo_etl.Load_Data.Save_to_DB manual__2022-02-19T16:21:55.210923+00:00 [queued]>
[2022-02-19 16:22:30,868] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-02-19 16:22:30,868] {taskinstance.py:1239} INFO - Starting attempt 1 of 1
[2022-02-19 16:22:30,869] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-02-19 16:22:31,008] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): Load_Data.Save_to_DB> on 2022-02-19 16:21:55.210923+00:00
[2022-02-19 16:22:31,044] {standard_task_runner.py:52} INFO - Started process 42616 to run task
[2022-02-19 16:22:32,070] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'demo_etl', 'Load_Data.Save_to_DB', 'manual__2022-02-19T16:21:55.210923+00:00', '--job-id', '405', '--raw', '--subdir', 'DAGS_FOLDER/demo_etl.py', '--cfg-path', '/tmp/tmpsderntk_', '--error-file', '/tmp/tmpu3x4ztbs']
[2022-02-19 16:22:32,071] {standard_task_runner.py:77} INFO - Job 405: Subtask Load_Data.Save_to_DB
[2022-02-19 16:22:37,465] {logging_mixin.py:109} INFO - Running <TaskInstance: demo_etl.Load_Data.Save_to_DB manual__2022-02-19T16:21:55.210923+00:00 [running]> on host d18a6d5a39bc
[2022-02-19 16:22:47,281] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=demo_etl
AIRFLOW_CTX_TASK_ID=Load_Data.Save_to_DB
AIRFLOW_CTX_EXECUTION_DATE=2022-02-19T16:21:55.210923+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-02-19T16:21:55.210923+00:00
[2022-02-19 16:22:47,345] {logging_mixin.py:109} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152 AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
[2022-02-19 16:22:47,346] {logging_mixin.py:109} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152 AirflowContextDeprecationWarning: Accessing 'next_ds' from the template is deprecated and will be removed in a future version. Please use '{{ data_interval_end | ds }}' instead.
[2022-02-19 16:22:47,464] {logging_mixin.py:109} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152 AirflowContextDeprecationWarning: Accessing 'next_ds_nodash' from the template is deprecated and will be removed in a future version. Please use '{{ data_interval_end | ds_nodash }}' instead.
[2022-02-19 16:22:47,465] {logging_mixin.py:109} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152 AirflowContextDeprecationWarning: Accessing 'next_execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_end' instead.
[2022-02-19 16:22:47,467] {logging_mixin.py:109} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152 AirflowContextDeprecationWarning: Accessing 'prev_ds' from the template is deprecated and will be removed in a future version.
[2022-02-19 16:22:47,467] {logging_mixin.py:109} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152 AirflowContextDeprecationWarning: Accessing 'prev_ds_nodash' from the template is deprecated and will be removed in a future version.
[2022-02-19 16:22:47,500] {logging_mixin.py:109} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152 AirflowContextDeprecationWarning: Accessing 'prev_execution_date' from the template is deprecated and will be removed in a future version.
[2022-02-19 16:22:47,501] {logging_mixin.py:109} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152 AirflowContextDeprecationWarning: Accessing 'prev_execution_date_success' from the template is deprecated and will be removed in a future version. Please use 'prev_data_interval_start_success' instead.
[2022-02-19 16:22:47,581] {logging_mixin.py:109} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152 AirflowContextDeprecationWarning: Accessing 'tomorrow_ds' from the template is deprecated and will be removed in a future version.
[2022-02-19 16:22:47,583] {logging_mixin.py:109} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152 AirflowContextDeprecationWarning: Accessing 'tomorrow_ds_nodash' from the template is deprecated and will be removed in a future version.
[2022-02-19 16:22:47,588] {logging_mixin.py:109} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152 AirflowContextDeprecationWarning: Accessing 'yesterday_ds' from the template is deprecated and will be removed in a future version.
[2022-02-19 16:22:47,589] {logging_mixin.py:109} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:152 AirflowContextDeprecationWarning: Accessing 'yesterday_ds_nodash' from the template is deprecated and will be removed in a future version.
[2022-02-19 16:22:53,895] {logging_mixin.py:109} INFO - Done !!
[2022-02-19 16:23:04,303] {logging_mixin.py:109} INFO - Done !!
[2022-02-19 16:23:04,541] {logging_mixin.py:109} INFO -       vendorid     lpep_pickup_datetime  ... trip_type  congestion_surcharge
0            2  2019-12-18T15:52:30.000  ...         1                  0.00
1            2  2020-01-01T00:45:58.000  ...         2                  0.00
2            2  2020-01-01T00:41:38.000  ...         1                  0.00
3            1  2020-01-01T00:52:46.000  ...         1                  2.75
4            1  2020-01-01T00:19:57.000  ...         1                  0.00
...        ...                      ...  ...       ...                   ...
1995         2  2021-01-02T12:48:43.000  ...         1                  0.00
1996         1  2021-01-02T12:16:47.000  ...         1                  0.00
1997         2  2021-01-02T12:07:17.000  ...         1                  0.00
1998         2  2021-01-02T12:32:45.000  ...         1                  0.00
1999         2  2021-01-02T12:02:46.000  ...         1                  0.00

[2000 rows x 18 columns]
[2022-02-19 16:23:07,512] {logging_mixin.py:109} INFO - Save Data to Database Done !!
[2022-02-19 16:23:07,512] {python.py:175} INFO - Done. Returned value was: None
[2022-02-19 16:23:07,528] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=demo_etl, task_id=Load_Data.Save_to_DB, execution_date=20220219T162155, start_date=20220219T162230, end_date=20220219T162307
[2022-02-19 16:23:07,599] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-02-19 16:23:07,663] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
